### ISSUE TITLE
Discussion: can one store the md5 encoded version of the password in odyssey.conf ?

### ISSUE THREAD

**Issue by rechtem** [yandex/odyssey#265]

**Comment by rechtem** [comment]
Setting authentication "md5", I tried to encode the password, but only a clear password makes the connection.
Is there a way to md5 encode the password and store the encoded version in the conf file ?
Would it be possible to not set a password at all and do some kind of pass-through from psql client to the rdbms ?

**Comment by x4m** [comment]
This works
```
database "db1" {
    user "user1" {
        authentication "md5"
        password "md5a5a5a5a5a5a5a5a5a5a"
        storage "pg_server"
....
    }
```

**Comment by rechtem** [comment]
OK thanks. 

The problem is that I misunderstood these parameters: they actually govern the connection between the end user client (psql) and Odyssey, but not the connection between Odyssey and PG server. For the later, one has to use storage_user and storage_password. This implies that storage_password has to be put in plain text in the conf file.

Would not be it possible to implement a pass-through mode to avoid such issue ?

**Comment by reshke** [comment]
yes this is a good feature i think we should do it. I can't say about the timing, but maybe I or Andrey will do it in spare time ...

**Comment by reshke** [comment]
also PR are welcome ;)
